tests/console/perl_bootloader.pm

```diff
@@ -20,17 +19,38 @@ use version_utils 'is_sle';
 sub run {
     my ($self) = @_;
     select_serial_terminal;
+    zypper_call 'in perl-Bootloader';
 
-    if (script_run 'rpm -q perl-Bootloader' == 1) {
-        zypper_call 'in perl-Bootloader';
+    # Set bootloader to grub2 and check if it was changed in /etc/sysconfig/bootloader
+    if (is_sle(">=15-SP3")) {
+        assert_script_run 'pbl --loader grub2';
+        validate_script_output 'cat /etc/sysconfig/bootloader | grep "LOADER_TYPE"', sub { m/grub2/ };
     }
-    assert_script_run 'mkinitrd';
-    assert_script_run 'update-bootloader';
+    assert_script_run 'pbl --install';
+    assert_script_run 'pbl --config';
 
-    if (is_sle('>=12-SP2')) {
-        validate_script_output('pbl --show', sub { /grub2/ });
+    # Add new option and check if it exists
+    assert_script_run 'pbl --add-option TEST_OPTION="test_value"';
+    if (is_sle(">=15-SP3")) {
+        validate_script_output 'pbl --get-option "TEST_OPTION"', sub { m/test_value/ };
```

Comment:
Is the option no longer set anywhere else? Testing whether pbl itself has it is IMO useless